app/src/main/java/com/github/se/studybuddies/ui/timer/TimerScreenContent.kt

```diff
@@ -32,21 +32,25 @@ import com.github.se.studybuddies.ui.theme.Blue
 import com.github.se.studybuddies.ui.theme.White
 import com.github.se.studybuddies.viewModels.TimerViewModel
 
+@SuppressLint("SuspiciousIndentation")
 @Composable
 fun TimerScreenContent(timerViewModel: TimerViewModel, navigationActions: NavigationActions) {
-  val timerValue by timerViewModel.timer.collectAsState()
+  val timerValue = timerViewModel.timerValue.collectAsState()
   val timerEnd by timerViewModel.timerEnd.collectAsState()
-
-  TimerScreen(
-      navigationActions = navigationActions,
-      timerValue = timerValue,
-      timerEnd = timerEnd,
-      onAddHours = { hours: Long -> timerViewModel.addHours(hours) },
-      onAddMinutes = { minutes: Long -> timerViewModel.addMinutes(minutes) },
-      onAddSeconds = timerViewModel::addSeconds,
-      onStart = { timerViewModel.startTimer() },
-      onPause = { timerViewModel.pauseTimer() },
-      onReset = { timerViewModel.resetTimer() })
+  timerValue?.let {
```

Comment:
Can delete those two null-safe accesses as they won't ever be null, above and below this comment

Reply:
it is for safety reasons they won't let me delete it